### PR TITLE
PT-879 Create Parameter object in DeviceData

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: docs/deviceParameter.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+List of SubTypes used to process the parameter.
+
+```
+QUICK SUMMARY
+Required:
+    platform: if processed = `yes`
+Range:
+    `yes`
+    `no`
```
Comment (CedricPerotto) [inline docs/deviceParameter.md]: Copy paste typo for the Range ?

File: docs/deviceParameter.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+### units
+
+Units that is used for this parameter. 
+
+```
+QUICK SUMMARY
+Required:
+    platform: yes
```
Comment (CedricPerotto) [inline docs/deviceParameter.md]: Some parameters don't have any unit but the field is required ? 
Maybe we should have a default value when unit is N/A.

File: docs/deviceParameter.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+### minValue 
+
+Minimal value of the device parameter. 
+
+```
+QUICK SUMMARY
+Required:
+    platform: yes
```
Comment (CedricPerotto) [inline docs/deviceParameter.md]: We always have a minValue/maxValue for all the parameters ? 
We may have "string" typed parameters that won't handle min/max values.

File: docs/deviceParameter.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+### maxValue 
+
+Maximal value of the device parameter. 
+
+```
+QUICK SUMMARY
+Required:
+    platform: yes
```
Comment (CedricPerotto) [inline docs/deviceParameter.md]: We always have a minValue/maxValue for all the parameters ? 
We may have "string" typed parameters that won't handle min/max values.

File: docs/deviceParameter.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  "timezoneOffset": 60,
+  "name": "Name of the Parameter",
+  "value": "Current value of the parameter",
+  "units": "%",
+  "lastUpdateDate": "2020-01-20T08:10:00.000Z",
+  "previousValue": "Previous value of the parameter",  
+  "level": "1",
+  "minValue": "0",
+  "maxValue": "10",
+  "processed": "True",
```
Comment (CedricPerotto) [inline docs/deviceParameter.md]:
```suggestion
  "processed": "yes",
```

File: docs/deviceParameter.md (hunk 1 above)
Comment (franck-fourel) [reply docs/deviceParameter.md]: yep

**Review by BigFab — APPROVED** [review]
LGTM :)

**Pull request merged**